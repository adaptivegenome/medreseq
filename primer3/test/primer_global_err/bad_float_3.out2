../src/primer3_core: Illegal PRIMER_MAX_SELF_END value: 
