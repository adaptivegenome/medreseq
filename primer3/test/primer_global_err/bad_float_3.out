SEQUENCE_ID=bad_float
SEQUENCE_TEMPLATE=GTAGTCAGTAGACNATGACNACTGACGATGCAGACNACACACACACACACAGCACACAGGTATTAGTGGGCCATTCGATCCCGACCCAAATCGATAGCTACGATGACG
PRIMER_MAX_SELF_END=
PRIMER_PRODUCT_SIZE_RANGE=75-100
PRIMER_PICK_INTERNAL_OLIGO=1
SEQUENCE_INTERNAL_EXCLUDED_REGION=37,21
PRIMER_ERROR=Illegal PRIMER_MAX_SELF_END value: 
=
